azure-nextgen-ts-webapp-privateendpoint-vnet-injection/index.ts

```diff
@@ -0,0 +1,174 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as azure_nextgen from "@pulumi/azure-nextgen";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+
+const config = new pulumi.Config();
+const resourceGroupNameParam = config.require("resourceGroupNameParam");
```

Comment:
Why require the resource group name? Should we add a default? Otherwise, we need add a command to set it to the readme.

Reply:
Added a default and cleaned up a lot of the parameter cruft from the arm template conversion.